iterableapi/src/main/java/com/iterable/iterableapi/IterableUtil.java

```
@@ -8,6 +8,8 @@
 import androidx.annotation.Nullable;
 import androidx.annotation.VisibleForTesting;
 
+import com.google.android.gms.common.util.IOUtils;
```

Is this public API? Is it possible that this might change?

---

Looks like it is not documented here:
https://developers.google.com/android/reference/com/google/android/gms/common/package-summary
I'd be cautious using this in a library since an update to `play-services-basement` might break our artifact at runtime.

---

Basically what that one does is to wrap the close in a try-catch and fail silently if for whatever reason the close throws. Would you prefer a manual implementation of that instead?

---

Yeah, if it's a few lines of code, I'd prefer to move it to one of our util classes.